kubernetes/manifest-templates/velum.yaml

```diff
@@ -82,3 +90,6 @@ spec:
     - name: velum-source-code
       hostPath:
         path: ${project_dir}
+    - name: velum-secrets
+      hostPath:
+        path: /tmp/velum-secrets
```

Comment:
I think `/tmp` dir is cleanup using various mechanisms. If the host decides to cleanup its /tmp dir or the guest decides to do the same, then bye bye secrets. If that happens, all active sessions will become invalid, no? Do we want to persist the secrets across reboots and whatever? If yes then maybe `/tmp` is not the best choice (both on host and guest).